src/courses/advanced/09.md

```diff
@@ -218,28 +223,32 @@ jobs:
        # path to our profile                   
       PROFILE: my_nginx     
 ```
+
 :::
 
 - `gold-image` is an arbitrary name we gave this job. It would be more useful if we were running more than one.
 - `name` is a simple title for this job.
 - `runs-on` declares what operating system we want our runner node to be. We picked Ubuntu (and we suggest you do to to make sure the rest of the workflow commands work correctly).
 - `env` declares environment variables for use by any step of this job. We will go ahead and set a few variables for running InSpec later on:
-    - `CHEF_LICENSE` will automatically accept the license prompt when you run InSpec the first time so that we don' hang waiting for input!
-    - `PROFILE` is set to the path of the InSpec profile we will use to test. This will make it easier to refer to the profile multiple times and still make it easy to swap out.
+  - `CHEF_LICENSE` will automatically accept the license prompt when you run InSpec the first time so that we don' hang waiting for input!
```

Comment:
```suggestion
  - `CHEF_LICENSE` will automatically accept the license prompt when you run InSpec the first time so that we don't hang waiting for input!
```